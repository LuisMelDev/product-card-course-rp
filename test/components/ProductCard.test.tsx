import React from 'react';
import renderer from 'react-test-renderer';
import { ProductCard } from '../../src/components';

describe('it', () => {
  it('should render correcly', () => {
    const wrapper = renderer.create(
      <ProductCard
        product={{
          id: '1',
          title: 'Product 1',
        }}
      >
        {() => <div>Hola mundo</div>}
      </ProductCard>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
