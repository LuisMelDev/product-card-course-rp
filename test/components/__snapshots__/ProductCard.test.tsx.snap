// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render correcly 1`] = `
<div
  className="productCard undefined"
>
  <div>
    Hola mundo
  </div>
</div>
`;
